braid/braid_status.h

```diff
@@ -654,20 +681,22 @@ ACCESSOR_HEADER_GET1(Objective, Tol,           Real)
  */
 
 /** When CallingFunction equals 0, Braid is in FInterp */
-#define braid_ASCaller_FInterp   0
-/** When CallingFunction equals 0, Braid is in FRestrict */
+#define braid_ASCaller_FInterp 0
+/** When CallingFunction equals 1, Braid is in FRestrict */
 #define braid_ASCaller_FRestrict 1
-/** When CallingFunction equals 0, Braid is in FRefine */
-#define braid_ASCaller_FRefine   2
-/** When CallingFunction equals 0, Braid is in FAccess */
-#define braid_ASCaller_FAccess   3
+/** When CallingFunction equals 2, Braid is in FRefine */
+#define braid_ASCaller_FRefine 2
+/** When CallingFunction equals 3, Braid is in FAccess */
+#define braid_ASCaller_FAccess 3
 /** When CallingFunction equals 4, Braid is inside FRefine after the new finest
  * level has been initialized */
-#define braid_ASCaller_FRefine_AfterInitHier   4
+#define braid_ASCaller_FRefine_AfterInitHier 4
 /** When CallingFunction equals 5, Braid is at the top of the cycle */
-#define braid_ASCaller_Drive_TopCycle   5
+#define braid_ASCaller_Drive_TopCycle 5
 /** When CallingFunction equals 6, Braid is in FCrelax */
 #define braid_ASCaller_FCRelax 6
+/** When CallingFunction equals 6, Braid just finished initialization */
```

Comment:
This should be "... equals 7, Braid just finished initialization */ "
Thanks for fixing those typos above!